### PR TITLE
chore: remove non working vault closing code

### PR DESCRIPTION
**Pull request by carlos-affinidi**

### CODE REVIEW

File: libs/iota-browser/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@affinidi-tdk/iota-browser",
-  "version": "0.2.4",
+  "version": "0.2.5",
```
Comment (maratsh) [inline libs/iota-browser/package.json]: **note:** 
versioning has to be done by semantic-release

File: libs/iota-browser/affinidi-tdk-iota-browser-0.2.4.tgz
Comment (maratsh) [inline libs/iota-browser/affinidi-tdk-iota-browser-0.2.4.tgz:1]: **issue:** 
Why do we put binaries in repo ? We have standard release process to npm in place

**Review by maratsh — CHANGES_REQUESTED** [review]

File: libs/iota-browser/package.json (hunk 1 above)
Comment (carlos-affinidi) [reply libs/iota-browser/package.json]: yeah, we will reset before merging

File: libs/iota-browser/affinidi-tdk-iota-browser-0.2.4.tgz
Comment (carlos-affinidi) [reply libs/iota-browser/affinidi-tdk-iota-browser-0.2.4.tgz:1]: We will remove it before merging and the package is published. Having it here for now makes it easier to use for now.

**Pull request merged**